pipeline/terraform0.12/stack/vhs.tf

```diff
@@ -0,0 +1,6 @@
+// TODO: (tf0.12 upgrade) upgrade this
```

Comment:
Is this a TODO you've added, or one we should now address?

Reply:
Later as then we can look at that logic separately - PRs shouldn't be large to force us not to be lazy.